### ISSUE TITLE
Gurobashi arena not spawning correct. 

### ISSUE THREAD

**Issue by FroggeRa** [Looking4Group/L4G_Core#3731]
I was here at arena 20:40-to now 21:20 no chest spawn at all comeon guys u have to step up now

**Comment by abuescher** [comment]
Yeah you really have to step up, arena object not spawning is gamebreaking for tbc expansion.

**Comment by FroggeRa** [comment]
2 just found 2 bugs and yes one of them are in outland lol

**Comment by AnonXS** [comment]
afaik the time is delayed by 1hour.
